backend/tests/test_reservation_routes.py

```
@@ -96,3 +121,101 @@ def test_get_reservations(client):
+    invalid_structure_id = 9999  # Assuming this structure ID doesn't exist
+
+    response = client.get(f"/api/v1/reservations/monthly/{invalid_structure_id}")
+    
+    assert response.status_code == 404
+    data = response.get_json()
+    assert data["message"] == "Structure not found"  # Ensure the message matches the error returned
+
+    """Test when no reservations exist for a structure."""
+    """ _, structure_id = init_db  # DB is clean from fixture """
+
+    response = client.get(f"/api/v1/reservations/monthly/1")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)